### PR TITLE
Add code examples to the README

### PR DESCRIPTION
**Pull request by PineappleIOnic**
This PR adds code examples and explanations to the README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,6 +27,335 @@ use Utopia\Orchestration\Orchestration;
...
+
+        The ID of the container to connect to the network.
+
+    - `network_id` [String] [Required]
+
+        The ID of the network to connect to.
+
+    </details>
+
+- ### Disconnect a container from a network
```
Comment (Meldiron) [inline README.md:330]: In preview, this looks weird, but I can't see a problem... Can you?

CleanShot 2022-03-22 at 08 59 17@2x

Comment (eldadfux) [reply README.md:330]: Maybe because it's under a bullet list?

Comment (PineappleIOnic) [reply README.md]: Fixed 👍

File: README.md [hunk 2]
```diff
@@ -27,6 +27,362 @@ use Utopia\Orchestration\Orchestration;
...
+    </summary>
+    <br>
+
+    - `name` [String] [Required]
+
+        The name of the network.
+
+    - `internal` [Boolean]
+
+        Whether to set the network to be an internal network.
```
Comment (Meldiron) [inline README.md:309]: No idea what internal network means.. Is this our term, or coming from Docker? If from Docker, we don't need to explain it.

File: README.md [hunk 3]
```diff
@@ -27,6 +27,362 @@ use Utopia\Orchestration\Orchestration;
...
+
+        The ID of the container to disconnect from the network.
+
+    - `network_id` [String] [Required]
+
+        The ID of the network to disconnect from.
+
+    - `force` [Boolean]
+
+        Whether to force disconnect the container.
```
Comment (Meldiron) [inline README.md:381]:
```suggestion
        Whether to disconnect the container forcefully.
```

**Review by Meldiron — APPROVED** [review]
Looks good to me, great detailed docs 💪

File: README.md [hunk 4]
```diff
@@ -15,18 +15,364 @@ Install using composer:
...
 
-Init in your application:
+### Example
 ```php
 <?php
 
-require_once __DIR__ . '/../../vendor/autoload.php';
+require_once 'vendor/autoload.php';
 
 use Utopia\Orchestration\Orchestration;
```
Comment (christyjacob4) [inline README.md:24]: Screenshot 2022-03-25 at 3 40 14 PM

Not an issue with your PR but I noticed a typo. So please fix it as well

**Review by christyjacob4 — APPROVED** [review]

File: README.md (hunk 4 above)
Comment (PineappleIOnic) [reply README.md:24]: Fixed 👍

**Pull request merged**